Composer/packages/types/src/publish.ts

```diff
@@ -14,9 +14,25 @@ export type PublishResult = {
   eTag?: string;
   log?: string;
   id?: string;
-  time?: Date;
+  time?: string;
   endpointURL?: string;
   status?: number;
+  action?: {
+    href: string;
+    label: string;
+  } | null;
+};
+
+export type ProcessStatus = {
+  id: string; // unique id
```

Comment:
It will be better if put the comments above this line to leverage IDE's intellisense feature.
```
/** Unique Process Id */
id: string;
```